packages/config/config/devices/0x0344/he-ht01.json

```diff
@@ -0,0 +1,795 @@
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Frequency Region",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 9,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "EU (868.4 MHz, 869.85 MHz)",
+					"value": 0
+				},
+				{
+					"label": "US (908.4 MHz, 916.0 MHz)",
+					"value": 1
+				},
+				{
+					"label": "AU (919.8 MHz, 921.4 MHz)",
+					"value": 2
+				},
+				{
+					"label": "HO (919.8 MHz)",
+					"value": 3
+				},
+				{
+					"label": "In (865.2 MHz)",
+					"value": 4
+				},
+				{
+					"label": "IL (916.0 MHz)",
+					"value": 5
+				},
+				{
+					"label": "RU (869.0 MHz)",
+					"value": 6
+				},
+				{
+					"label": "Cn (868.4 MHz)",
+					"value": 7
+				},
+				{
+					"label": "JP (922.5 MHz, 923.9 MHz, 926.3 MHz)",
+					"value": 8
+				},
+				{
+					"label": "Or (920.9 MHz, 921.7 MHz, 923.1 MHz)",
+					"value": 9
+				}
+			]
+		},
+		"3": {
+			"label": "Hardware and Software Versions",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 0,
+			"readOnly": true
+		},
+		"4": {
+			"label": "Energy Consumption",
+			"unit": "kW",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 0,
+			"readOnly": true
+		},
+		"5": {
+			"label": "Display Brightness Control",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"defaultValue": 0
+		},
+		"6": {
+			"label": "Touch Buttons Sensitivity",
+			"description": "Allowable range: 1 (low sensitivity) to 10 (high sensitivity).",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 10,
+			"defaultValue": 6
+		},
+		"7": {
+			"label": "Relay Output Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "NO",
+					"value": 0
+				},
+				{
+					"label": "NC",
+					"value": 1
+				}
+			]
+		},
+		"8": {
+			"label": "External Input Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Ignore external input",
+					"value": 0
+				},
+				{
+					"label": "Toggle Switch",
+					"value": 1
+				},
+				{
+					"label": "Toggle Switch Reverse",
+					"value": 2
+				},
+				{
+					"label": "Push Button (Momentary Switch)",
+					"value": 3
+				}
+			]
+		},
+		"9": {
+			"label": "Operating Mode for External Input Action",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 6,
+			"defaultValue": 6,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "COM",
+					"value": 1
+				},
+				{
+					"label": "TIME",
+					"value": 2
+				},
+				{
+					"label": "DRY",
+					"value": 3
+				},
+				{
+					"label": "ECO",
+					"value": 4
+				},
+				{
+					"label": "VAC",
+					"value": 5
+				},
+				{
+					"label": "MAN",
+					"value": 6
+				}
+			]
+		},
+		"10": {
+			"label": "Floor Sensor Resistance",
+			"unit": "kohm",
```

Comment:
You used the Ohm symbol above, which I find preferable

Reply:
fixed